downstream/modules/dev-guide/con-content-workflows.adoc

```diff
@@ -14,5 +14,5 @@ As {ControllerName} shifts to using {ExecEnvName}, tools like {Navigator} and {B
 [role="_additional-resources"]
 .Additional resources
 
-* See the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/ansible_navigator_creator_guide/index[Automation Content Navigator Creator Guide] for more on using {Navigator}.
+* See the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_creator_guide/index#con-about-ansible-navigator_tools[Automation Content Navigator Creator Guide] for more on using {Navigator}.
```

Comment:
Not a problem with this one, but wondered if you used html-single on purpose? The user could still get there using the multi-page view (https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html/red_hat_ansible_automation_platform_creator_guide/tools#con-about-ansible-navigator_tools). This is not a suggestion to change this. Just wondering if you selected the single view purposely.

Reply:
oh good point, I'll change it.